### ISSUE TITLE
Option to search by file path

### ISSUE THREAD

**Issue by P4tela** [ForkIssues/Tracker#1823]
Hi there!
Big fan of Fork, been using it for over 1 year and I love it!
Today I did a oopsie in my job... I deleted a file from my repository a while back (thinking it was trash) and today i needed it because a client of mine deleted that file from his computer ( This is a old language and client so that's why we have local files) and I was looking for it in my laptop and I couldn't find it. I searched it in fork and I managed to find it! Only BECAUSE the first time i commited it, his name was on the description... So the search worked and everything was saved (thank you btw :) ). However I wanted to ask if it's not possible to add a new search option, where you search for a specific file and not the description or subject. I know that would take a lot of time, but if for some reason I brilliantly manage do this again in the future, this new search option will "save" me!

If this feature already exists then I'm sorry. I tried to look for it and couldn't find it... If it does exist please let me know how I would be able to do that...

TL;DR:New feature where you search for a file instead of subject or description.

**Comment by DanPristupov** [comment]
Hi.

Thank you for the kind words about Fork :)

I think you can search by file path:
<img width="258" alt="Screenshot 2023-02-23 at 8 55 58" src="https://user-images.githubusercontent.com/618115/220850316-f8353b4b-6e37-4b40-8a19-71632d0335bc.png">

You can also search by a piece of source code:
<img width="260" alt="Screenshot 2023-02-23 at 8 56 33" src="https://user-images.githubusercontent.com/618115/220850394-5629b063-3748-4f97-b5cf-ceebdd9a15fb.png">

**Comment by P4tela** [comment]
Hey again! It seems i was indeed being dumb :)
I tried every search option (including "Diff Content") and it doesn't work in the situation I described. 
I tried again on the "Path" option but instead of just putting the file name, i tried with "asterisk/fileName" and it worked like a charm ("asterisk" because I wouldn't know the original file location).

Thank you again! I will close this :)
